### ISSUE TITLE
Customize labels

### ISSUE THREAD

**Issue by kikky7** [thedevdojo/auth#98]
**Is your feature request related to a problem? Please describe.**
Ability to customize labels the same way as layout and other similar settings.

**Describe the solution you'd like**
For example position label above input, below input, instead over the top border, change font size, etc.

**Describe alternatives you've considered**
Override via css.

**Comment by skdishansachin** [comment]
So, you can totally tweak the CSS. In the authentication setup, head over to the appearance section and pick the custom CSS tab. That's where you can write your own CSS. But if you want more control over the styles, you should publish the components and customize them as you like. Check out the documentation for that https://devdojo.com/auth/docs/setup-customizations/.

**Comment by kikky7** [comment]
Yes, component override would be much better approach in case there are more updates around the layout.

I didn't find the docs about it.
